cmd/porter/bundle.go

```diff
@@ -81,6 +82,8 @@ Porter uses the docker driver as the default build driver, an alternate driver m
 		"Path to the build context directory where all bundle assets are located.")
 	f.StringVar(&opts.Driver, "driver", porter.BuildDriverDefault,
 		fmt.Sprintf("Experimental. Driver for building the invocation image. Allowed values are: %s", strings.Join(porter.BuildDriverAllowedValues, ", ")))
+	f.StringSliceVar(&opts.Customs, "custom", nil,
+		"Define an individual key-value pair for custom section in the form of NAME=VALUE. May be specified multiple times.")
```

Comment:
Maybe we should include how to set a nested value?
```suggestion
		"Define an individual key-value pair for the custom section in the form of NAME=VALUE. Use dot notation to specify a nested custom field.  May be specified multiple times.")
```